backend/tracim_backend/command/cleanup.py

```diff
@@ -0,0 +1,222 @@
+import argparse
+import typing
+
+from pyramid.scripting import AppEnvironment
+
+from tracim_backend import UserDoesNotExist
+from tracim_backend.command import AppContextCommand
+from tracim_backend.lib.cleanup.cleanup import CleanupLib
+from tracim_backend.lib.core.user import UserApi
+from tracim_backend.models.auth import User
+from tracim_backend.models.tracim_session import unprotected_content_revision
+
+
+class DeleteResultIds(object):
+    def __init__(self, user_id: int, workspace_ids: typing.List[int] = None):
+        self.user_id = user_id
+        self.workspace_ids = workspace_ids or []
+
+
+class DeleteUserCommand(AppContextCommand):
+    def get_description(self) -> str:
+        return """Remove user and associated information from database"""
+
+    def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
+        parser = super().get_parser(prog_name)
+        parser.add_argument(
+            "-best-effort",
+            "--best-effort",
+            help="anonymise if needed user",
```

Comment:
This description is little obscure i think.
```suggestion
            help="anonymizes the user where he cannot be deleted",
```